aucoalesce/testdata/ubuntu-16.10-linux-4.8.0.json.golden

```diff
@@ -428,33 +531,32 @@
           "uid": "0"
         }
       },
-      "action": "accepted-connection-from",
-      "thing": {
-        "primary": "72.83.230.100",
-        "secondary": "58140",
-        "what": "socket"
+      "process": {
+        "pid": "1663",
+        "ppid": "1",
+        "title": "(sshd)",
+        "name": "sshd",
+        "exe": "/usr/sbin/sshd"
+      },
+      "source": {
+        "ip": "72.83.230.100",
+        "port": "58140"
+      },
+      "network": {
+        "direction": "incoming"
       },
-      "how": "/usr/sbin/sshd",
-      "key": "net",
       "data": {
         "a0": "3",
         "a1": "7ffd0dc80040",
         "a2": "7ffd0dc7ffd0",
         "a3": "0",
         "arch": "x86_64",
-        "comm": "sshd",
-        "exe": "/usr/sbin/sshd",
         "exit": "5",
-        "pid": "1663",
-        "ppid": "1",
-        "proctitle": "(sshd)",
+        "socket_addr": "72.83.230.100",
+        "socket_family": "ipv4",
+        "socket_port": "58140",
         "syscall": "accept",
         "tty": "(none)"
-      },
-      "socket": {
```

Comment:
You had a namespace here :-)

Reply:
Well since I create the `source.ip` and `destination.ip` I felt like having a separate top-level `socket` field was redundant. In Auditbeat I can do something like `auditd.data.socket.addr` with the `data.socket_*` fields.